packages/muelu/adapters/xpetra/MueLu_RefMaxwell_def.hpp

```diff
@@ -171,34 +171,6 @@ namespace MueLu {
     FindNonZeros<Scalar,LocalOrdinal,GlobalOrdinal,Node>(myColsToZero->getData(0),dirichletCols);
   }
 
-
-  template<class Scalar, class LocalOrdinal, class GlobalOrdinal, class Node>
-  void ApplyRowSumCriterion(const Xpetra::Matrix<Scalar,LocalOrdinal,GlobalOrdinal,Node>& A,
```

Comment:
The Kokkos View version should be moved as well.

Reply:
Done